libs/wingc/src/parser.rs

```diff
@@ -939,7 +939,21 @@ impl<'s> Parser<'s> {
 					))
 				}
 			}
-
+			"loop_range" => {
+				let inclusive = if expression_node.child_by_field_name("inclusive").is_some() {
+					Some(true)
+				} else {
+					Some(false)
+				};
+				Ok(Expr::new(
+					ExprKind::Range {
+						start: Box::new(self.build_expression(&expression_node.child_by_field_name("start").unwrap())?),
+						inclusive: inclusive,
+						end: Box::new(self.build_expression(&expression_node.child_by_field_name("end").unwrap())?),
```

Comment:
nit: we started to try and use more `expect` instead of `unwrap` in the compiler so maybe we can just swap it out with:

```suggestion
						start: Box::new(self.build_expression(&expression_node.child_by_field_name("start").expect("range expression should always include start"))?),
						inclusive: inclusive,
						end: Box::new(self.build_expression(&expression_node.child_by_field_name("end").expect("range expression should always include end"))?),
```